### PR TITLE
Random Menu- Lunch

### PR DESCRIPTION
**Pull request by wangg131**
I completed the Lunch level.

### CODE REVIEW

File: random_menu_lunch.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+puts "~Le Dinner Menu~"
+
+## three arrays for describing menu items
+adjective = ["fatty", "meaty", "decadent", "skinny",  "prickly", "room-temperature", "sultry", "heated", "dry", "moist"]
+cooking_method = ["fried", "impaled", "battered", "beaten", "tenderized", "smashed", "torched", "live", "infused", "vegan"]
+food = ["snake scales", "pandas", "Hagrid beards", "piglets", "tofurkey", "whale steak", "unicorn horns", "leprechauns", "pikachus", "walruses"]
+
+# I used a while loop to output meals equal to any number of elements in an array starting at 1
+meal_count = 1
+while meal_count <= adjective.length
```
Comment (kariabancroft) [inline random_menu_lunch.rb]: You could augment your code to use something like `[adjective.length, cooking_method.length, food.length].min` to get the smallest length from all of your lists to ensure that you aren't going too far in your loop if the lengths of each array are different.

File: random_menu_lunch.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+puts "~Le Dinner Menu~"
```
Comment (kariabancroft) [inline random_menu_lunch.rb:1]: I like the name of your menu :+1:

File: random_menu_lunch.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## three arrays for describing menu items
+adjective = ["fatty", "meaty", "decadent", "skinny",  "prickly", "room-temperature", "sultry", "heated", "dry", "moist"]
+cooking_method = ["fried", "impaled", "battered", "beaten", "tenderized", "smashed", "torched", "live", "infused", "vegan"]
+food = ["snake scales", "pandas", "Hagrid beards", "piglets", "tofurkey", "whale steak", "unicorn horns", "leprechauns", "pikachus", "walruses"]
+
+# I used a while loop to output meals equal to any number of elements in an array starting at 1
+meal_count = 1
+while meal_count <= adjective.length
+  puts "#{meal_count}. #{adjective.sample} #{cooking_method.sample} #{food.sample}" # called the .sample method on each array to choose an element randomly
```
Comment (kariabancroft) [inline random_menu_lunch.rb:11]: Great use of string interpolation and the sample method call

**Comment by kariabancroft** [comment]
Great job! The code is clean and concise. You could augment your logic a bit to determine the overall minimum value of the lengths of all of the arrays just in case they aren't the same.

**Comment by wangg131** [comment]
@kariabancroft Thanks Kari! I fixed it via your suggestion for my Lunch Menu. I was waiting to make/push changes until I had the chance to maybe attempt dinner, but this will do for now.

**Pull request merged**